### PR TITLE
Setup leap s06

### PR DESCRIPTION
**Pull request by DanielMwendwa**

### CODE REVIEW

File: configurations/leap_s06_pipeline.py [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+                        )
+                    ),
+                ],
+            ),
+        ],
+        ws_correct_dataset_code_scheme=load_code_scheme("ws_correct_dataset"),
+    ),
+    archive_configuration=ArchiveConfiguration(
+        archive_upload_bucket="gs://pipeline-execution-backup-archive",
+        bucket_dir_path="2022/POOL-WUSC/"
```
Comment (as2388) [inline configurations/leap_s06_pipeline.py:250]: I think we should break this down by season so it's easier to find archives for this season without having to download a few archives to check what's inside. e.g. '2022/POOL-WUSC/S06'

Comment (as2388) [inline configurations/leap_s06_pipeline.py:250]: I think if you have a trailing '/' the files will upload to `...POOL-WUSC//data-...`. The double '/' makes listing the contents a bit trickier (as an aside, we should probably update the relevant pipeline stage to trim trailing '/'s so we don't have to think about this in future)

**Review by as2388 — APPROVED** [review]

File: code_schemes/rqas/leap_s06/s06e06.json [hunk 2]
```diff
@@ -0,0 +1,205 @@
+{
+    "SchemeID": "Scheme-67596164",
+    "Name": "LEAP S06E06",
+    "Version": "0.0.0.1",
+    "Codes": [
+      {
+        "CodeID": "code-1a491a14",
```
[the remark this answers is not in the document]
Comment (DanielMwendwa) [reply code_schemes/rqas/leap_s06/s06e06.json]: updated.

File: configurations/leap_s06_pipeline.py (hunk 1 above)
Comment (DanielMwendwa) [reply configurations/leap_s06_pipeline.py]: updated

Comment (DanielMwendwa) [reply configurations/leap_s06_pipeline.py]: noted

File: configurations/leap_s06_pipeline.py [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+                                (36, 54): "36 to 54",
+                                (55, 99): "55 to 99"
+                            }
+                        )
+                    ),
+                ],
+            ),
+        ],
+        ws_correct_dataset_code_scheme=load_code_scheme("ws_correct_dataset"),
+    ),
```
[the remark this answers is not in the document]
Comment (DanielMwendwa) [reply configurations/leap_s06_pipeline.py:247]: [here](https://github.com/AfricasVoices/Pool-WUSC/pull/32)

**Pull request merged**